TEMPLATE.md

```diff
@@ -0,0 +1,91 @@
+---
+RFC: <PR number>
+Status: <Proposed>
+Version: <version number>
```

Comment:
I meant to say something like this :) (can be a date or anything else that identifies this particular instance of the template)
```suggestion
Version: 1.0
```